rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/SonarLintRpcClient.java

```diff
@@ -315,7 +315,7 @@ default void promoteExtraEnabledLanguagesInConnectedMode(PromoteExtraEnabledLang
 
   /**
    *  Called before every analysis to update the backend with analysis properties inferred by the client.
-   *  Example of such properties: sonar.java.* configurations, path to compile commands for CFamily analysis.
+   *  Example of such properties: sonar.java.* configurations
```

Comment:
If this comment is removed, I would expect to find it somewhere else. I know that we now expect the "path to compile commands" to be passed as part of user properties, but I think it could be confusing for clients. Also they would need to merge the path to compile commands with user-defined properties before calling the backend, so it's a bit more work for them. Have you considered having a separate notification only for the path?

Reply:
Added new notification for path to compile commands change. Analysis triggering logic moved there.